models/userDao.js

```diff
@@ -0,0 +1,69 @@
+const { appDataSource } = require("./dbconfig");
+
+const getHashedPassword = async (email) => {
+  try {
+    const hashedPassword = await appDataSource.query(
+      `SELECT
+                password AS hashedPassword
+            FROM
+                users
+            WHERE
+                email = ?;
+            `,
+      [email]
+    );
+    return hashedPassword;
+  } catch (err) {
+    console.log(err);
+    const error = new Error("GET_HASED_PASSWORD_FAILED");
+    error.statusCode = 401;
+    throw error;
+  }
+};
+
+const userEmail = async (email) => {
```

Comment:
```suggestion
const getUserEmail = async (email) => {
```
1. 엄격하게 정해진 규칙은 없지만, 동작 + 리소스의 조합으로 naming을 해주시면 보다 모든 코드에서 일관적이고 직관적이게 작성될 수 있습니다! (CRUD를 의미하는 create, get, update, delete + db에서 가져올 자원) 
2. parameter로 email을 받아서 return하는게 email이면 어떤 것을 위한 코드인가요?